src/logging.rs

```diff
@@ -53,8 +59,10 @@ impl<S: Subscriber> Layer<S> for CaptureLayer {
                 .send(LogEvent {
                     message,
                     metadata: event.metadata(),
+                    timestamp: OffsetDateTime::now_utc(),
                 })
-                .expect("LogEvents resource no longer exists!");
+                .inspect_err(|e| panic!("Failed to send log event: {}", e))
```

Comment:
i dont understand. `ok` is supposed to suppress an error and then we still panic if there was one? will `ok` ever do anything if it was `Err` before?

Reply:
it looks like `expect` was the right thing already